src/views/ajax.js

```diff
@@ -5,6 +5,36 @@ const arrowIcon = require('./open').arrowIcon;
 const icons = require('../assets/icons').default;
 const statusIcon = require('../assets/icons').statusIcon;
 
+/**
+ * removeOrigin - function to remove origin from a URL.
+ *
+ * @param {String} url URL to remove the origin from.
+ * @param {String} origin Origin to remove from URL.
+ */
+const removeOrigin = (url = '', origin = window.location.origin) => {
+  if (url.substr(0, origin.length) === origin) {
+      url = url.substr(origin.length, url.length);
+  }
+
+  return url;
+};
+
+/**
+ * removeOriginFromUrl - function to remove origin from a URL
+ *                       reguarding `http` and `https`.
+ *
+ * @param {String} url URL to remove the origin from.
+ * @param {String} origin Origin to remove from URL.
+ */
+const removeOriginFromUrl = (url = '', origin = window.location.origin) => {
```

Comment:
Not sure is this ever possible, in any case, we need to mock it to make sure it is there, but this is should be done with the tests work planned.